test/fixtures/cookbooks/test/recipes/plugins.rb

```diff
@@ -11,6 +11,10 @@
   action :install
 end
 
+grafana_plugin 'yesoreyeram-boomtable-panel' do
+  plugin_url 'https://github.com/axelrtgs/yesoreyeram-boomtable-panel/archive/feature/sorting.zip'
```

Comment:
Hmm, I'm not so confident about this being a magic it is just there thing (because one day you could remove it) can you stick a copy inside the cookbook and then on next pass we can flip it to grafana master